src/olympia/reviewers/models.py

```diff
@@ -755,6 +755,8 @@ class NeedsHumanReview(ModelBase):
     REASON_MANUALLY_SET_BY_REVIEWER = 7
     REASON_AUTO_APPROVED_PAST_APPROVAL_DELAY = 8
     REASON_ABUSE_REPORTS_THRESHOLD = 9
+    REASON_ABUSE_ADDON_VIOLATION = 11
+    REASON_ABUSE_ADDON_VIOLATION_APPEAL = 12
```

Comment:
Like https://github.com/mozilla/addons-server/pull/21376, those need to be added to the `choices` below. (Someday we should use `extended-choices` like we do in `AbuseReport` to get all of this for free here too...)